### PR TITLE
fix(log): respect log config

### PR DESCRIPTION
**Pull request by aymanbagabas**
Parse log config

### CODE REVIEW

File: server/git/git.go [hunk 1]
```diff
@@ -83,12 +83,12 @@ func RunGit(ctx context.Context, in io.Reader, out io.Writer, er io.Writer, dir
 	logger := log.FromContext(ctx).WithPrefix("rungit")
 	c := exec.CommandContext(ctx, "git", args...)
 	c.Dir = dir
-	c.Env = append(c.Env, envs...)
+	c.Env = append(os.Environ(), envs...)
 	c.Env = append(c.Env, "PATH="+os.Getenv("PATH"))
 	c.Env = append(c.Env, "SOFT_SERVE_DEBUG="+os.Getenv("SOFT_SERVE_DEBUG"))
 	if cfg != nil {
-		c.Env = append(c.Env, "SOFT_SERVE_LOG_FORMAT="+cfg.LogFormat)
-		c.Env = append(c.Env, "SOFT_SERVE_LOG_TIME_FORMAT="+cfg.LogTimeFormat)
```
Comment (caarlos0) [inline server/git/git.go:91]: q: why do we pass those envs down to `git`?

**Review by caarlos0 — APPROVED** [review]
cool, I noticed this too, was about to start working in it as well.

thanks!

File: server/git/git.go (hunk 1 above)
Comment (aymanbagabas) [reply server/git/git.go:91]: These envs get passed down to `git-receive-pack` and then to git-hooks like `pre-recieve` and `update`. Then in git-hooks, we call the soft-serve binary to run the hooks inside soft-serve.

Comment (caarlos0) [reply server/git/git.go:91]: Ahh gotcha! Thanks for explaining 🙏

**Pull request merged**